.env.example

```diff
@@ -3,7 +3,9 @@ HOSTNAME=localhost
 PORT=3000
 PROTOCOL=http
 REVERSE_PROXY_PORT=8080
-API_PATH=/api/v1/
+
+# Local storage configuration
+STORAGE_STATE_KEY=TAILOR_APP_STATE
```

Comment:
@ExtensionEngine/tailor Does this make sense? I wanted a central place for this to be defined so both client and server can be aware of it and the easiest approach was `.env`. That being said I'm not sure is there a value in this being configurable or should I find a better place for it to live and remain _hardcoded_?

Reply:
After offline discussion, I've decided to leave it hardcoded for now and remove it completely in the following PR.